uiowa/uiowa
# More bg-color combination work

Short term solutions in uids_base for https://github.com/uiowa/uids/issues/242 and https://github.com/uiowa/uids/issues/262

Fixes #2074 
Fixes #2005
Fixes #2080

## To Test

Try out the blocks (cta, collections, card, text area, body, webform, accordion, event and the footer contact block) with different color backgrounds and the gray text area block option.

You will also notice that SiteImprove extension no longer complains about #2074.
